templates/docker-compose/Makefile.dist

```diff
@@ -60,11 +60,12 @@ create_volumes:
 
 .PHONY: create_secrets
 create_secrets:
-	$(DOCKER_BIN) run -e SECRETS_DIR="$(SECRETS_DIR)" -v "$(SECRETS_DIR):/secrets" "public.ecr.aws/supportpal/helpdesk-configurator:$(CONFIGURATOR_VERSION)" sh "//app//scripts//create_secrets.sh"
+	$(DOCKER_COMPOSE_BIN) -f docker-compose.secrets.yml up --force-recreate
```

Comment:
Would `docker-compose run` work better? 🤔  https://docs.docker.com/compose/faq/#whats-the-difference-between-up-run-and-start

Reply:
Yep, I missed this

Reply:
I've changed it.